compiler/luci-ref-value-py-test/conftest.py

```diff
@@ -53,12 +54,20 @@ def copy_circle_model(model_src, model_dst):
     copy_if_changed(model_src, model_dst)
 
 
-def prepare_materials(test_name, tflrecipe_path, binary_path, artifacts_path):
-    ref_input_src = os.path.join(tflrecipe_path, test_name, 'ref.input')
+def prepare_materials(test_name, tflrecipe_path, circlerecipe, binary_path,
+                      artifacts_path):
+    # tfl? or circle?
+    recipe_path = tflrecipe_path
+    # check with 'test.recipe' file as 'ref.input?' can be absent for no input model
+    ref_input_src = os.path.join(recipe_path, test_name, 'test.recipe')
+    if (not os.path.isfile(ref_input_src)):
```

Comment:
`ref_input_src` seems not appropriate. How about renaming or removing the variable?

```suggestion
    tfl_recipe = os.path.join(recipe_path, test_name, 'test.recipe')
    if (not os.path.isfile(tfl_recipe)):
```
or

```suggestion
    
    if (not os.path.isfile(os.path.join(recipe_path, test_name, 'test.recipe'))):
```